label-inspector/label_inspector.ipynb

```diff
@@ -796,75 +794,861 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "real_time_preds = pd.read_csv(real_time_output_file)\n",
+    "real_time_preds.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Similar to above, the output contains two pieces of information:\n",
+    "\n",
+    "- the first column contains the predicted label for each example\n",
+    "- the remaining columns contain the predicted class probabilities for each example, these columns will have the names `[class_label]__probability`\n",
+    "    - in our sample dataset, the `A__probability` column shows the model predicted probability that an example should have the letter grade A"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 7. Clean Up Resources "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Delete models endpoints (skip if you skipped Section 6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you are done with performing real-time inferences, you no longer need the saved model and deployed endpoint. Here we show how to terminate the endpoint to avoid being charged."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# predictor.delete_model()\n",
+    "# predictor.delete_endpoint(delete_endpoint_config=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also delete the model associated with the batch transform job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# transformer.delete_model()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Delete S3 resources (optional)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The code cell below contains the code that can be used to automatically delete the files created in S3 in this sample notebook. Be wary that this command will delete all the files stored in the `base_folder_name` folder specified above. Proceed with caution if you have previously stored other files in that folder.\n",
+    "\n",
+    "Alternatively, you can navigate to the [S3 Management Console](https://s3.console.aws.amazon.com/) and manually delete these data files yourself."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# resource = session.resource('s3')\n",
+    "# resource_bucket = resource.Bucket(bucket_name)\n",
+    "\n",
+    "# # before executing the code below, ensure there is nothing else important in this folder\n",
+    "# resource_bucket.objects.filter(Prefix=f\"{base_folder_name}/\").delete()  "
+   ]
+  },
```

Comment:
maybe include a link at the very bottom user can click to see their S3 in aws console in order to verify themselves the files are deleted?

Reply:
text next to the link can say user can manually delete the files from there too

Reply:
I can't link directly to the bucket but the existing text just mentioned that they can check the s3 console:

> Alternatively, you can navigate to the [S3 Management Console](https://s3.console.aws.amazon.com/) and manually delete these data files yourself.